contrastsecurity/assets/dashboards/contrast_security_protect.json

```diff
@@ -0,0 +1,305 @@
+{
+  "board_title":"Contrast Security - Protect",
+  "read_only":false,
+  "description":null,
+  "created":"2019-11-07T16:17:55.348192+00:00",
+  "new_id":"8g4-chd-xe8",
+  "modified":"2019-11-07T16:17:55.348200+00:00",
+  "created_by":{
+    "disabled":false,
+    "handle":"kristiana.mitchell@contrastsecurity.com",
+    "name":"Kristiana Mitchell",
```

Comment:
This will appear on customer's instances. You may want to change it for something like:
```
"handle":"support@contrastsecurity.com",
"name":"Contrast Security",
...
"email":"support@contrastsecurity.com",
"icon":""
```
If you don't want your name and email to be that exposed.